tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -837,6 +837,46 @@ public:
    void AcceptVisitor(Detail::RFieldVisitor &visitor) const final;
 };
 
+template <>
+class RField<std::int64_t> : public Detail::RFieldBase {
```

Comment:
humm ... carefully here ... this is likely to lead to duplicate specialization (if 2 of the typedefs resolved to the same underlying type).

Reply:
In this case, are we OK by virtue of the guarantee that these are "signed integer type with width of exactly 8, 16, 32 and 64 bits respectively"? But it might be a problem for `std::int16_fast_t` "fastest signed integer type with width of at least 8, 16, 32 and 64 bits respectively" https://en.cppreference.com/w/cpp/types/integer